collectors/perf.plugin/metadata.yaml

```diff
@@ -1,54 +1,87 @@
+# yamllint disable rule:line-length
+---
 meta:
   plugin_name: perf.plugin
   module_name: perf.plugin
   monitored_instance:
     name: CPU performance
-    link: ''
+    link: "https://kernel.org/"
     categories:
-    - data-collection.linux-systems
-    icon_filename: 'bolt.svg'
+      - data-collection.linux-systems
+    icon_filename: "bolt.svg"
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
-    description: ''
-  keywords: []
+    description: ""
+  keywords:
+    - linux
+    - cpu performance
+    - cpu cache
+    - perf.plugin
   most_popular: false
 overview:
   data_collection:
-    metrics_description: 'Monitor CPU performance to ensure optimal computational operations. Monitor core usage, load averages, and thermal throttling for seamless computation tasks.'
-    method_description: ''
+    metrics_description: "This collector monitors CPU performance metrics about cycles, instructions, migrations, cache operations and more."
+    method_description: "It uses syscall (2) to open a file descriptior to monitor the perf events."
   supported_platforms:
-    include: []
+    include:
+      - Linux
     exclude: []
   multi_instance: true
   additional_permissions:
-    description: ''
+    description: "It needs setuid."
```

Comment:
Again, I am not sure we explain if this happens by default or if there should be another action that the user needs to take.

Reply:
Last commits address this.